### PR TITLE
[ISS-274] closes #274 to add updated verbiage requested by client

### LINKED ISSUE #274: Treemap Footnote and Heading Question Updates (similar to table)
#252 and #253 have information related to design.

For treemap visualization, include a text blurb somewhere (maybe at the bottom?) which explains what is being shown. 
The text could say: This visualization shows the occupations which {name of occupation} move to when they change occupation. The transition share is the proportion of {name of occupation} who move into a job in each other occupation when they switch occupation. We only break out individual occupations with transition shares greater than 0.2%.

For treemap visualization, include a text heading at the top which says “Which occupations do {name of occupation} move to?”
This is inspired by the treemap visualization for the Atlas of Economic Complexity, (https://atlas.cid.harvard.edu/explore), which has headers like “What does France export?”

### PR DESCRIPTION
**Pull request by jedpittman**
closes #274

### CODE REVIEW

**Comment by jedpittman** [comment]
@jkoren if you have time, and would not mind doing this review, I'd be grateful. I can merge if you review and approve it.

File: frontend/src/testing/data/transitionData.ts [hunk 1]
```diff
@@ -5,17 +5,31 @@ export default createTransitions([
   ['43-4051', 'Customer service representatives', 0.0421, 17.9, 37300],
   ['11-9051', 'Food service managers', 0.0407, 28.76, 59820],
   ['35-3011', 'Bartenders', 0.039, 13.46, 28000],
-  ['43-6014', 'Secretaries and administrative assistants, except legal, edical, and executive', 0.0391, 18.84, 39180,],
+  [
+    '43-6014',
+    'Secretaries and administrative assistants, except legal, edical, and executive',
```
Comment (jkoren) [inline frontend/src/testing/data/transitionData.ts:10]: should "edical" be "medical"?

File: frontend/src/testing/data/transitionData.ts [hunk 2]
```diff
@@ -5,17 +5,31 @@ export default createTransitions([
...
+    39180,
+  ],
   ['43-9061', 'Office clerks, general', 0.034, 17.48, 36360],
   ['41-2011', 'Cashiers', 0.0312, 11.72, 24370],
   ['11-9000', 'Other Management Occupations', 0.0303, 49.31, 102560],
-  ['35-1012', 'First-line supervisors of food preparation and erving workers', 0.0231, 17.77, 36960,],
-  ['41-1011', 'First-line supervisors of retail sales workers', 0.0199, 22.03, 45830,],
+  [
+    '35-1012',
+    'First-line supervisors of food preparation and erving workers',
```
Comment (jkoren) [inline frontend/src/testing/data/transitionData.ts:20]: should "erving" be "serving?"

**Review by jkoren — COMMENTED** [review]
made 2 comments on data

**Review by jkoren — APPROVED** [review]
made 2 comments on data

**Review by alexjball — APPROVED** [review]

File: frontend/src/testing/data/transitionData.ts (hunk 1 above)
Comment (jedpittman) [reply frontend/src/testing/data/transitionData.ts]: this is fixed in the latest version. Are we all set?

File: frontend/src/testing/data/transitionData.ts (hunk 2 above)
Comment (jedpittman) [reply frontend/src/testing/data/transitionData.ts]: this is fixed in the latest version. Are we all set?

**Pull request merged**